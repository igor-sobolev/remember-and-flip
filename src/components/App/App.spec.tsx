import * as React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { App } from '.';

describe('given App component', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  it('should match the snapshot', () => {
    expect(wrapper.getElement()).toMatchSnapshot();
  });
});
