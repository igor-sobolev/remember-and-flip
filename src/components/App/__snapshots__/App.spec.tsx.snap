// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`given App component should match the snapshot 1`] = `
<div
  className="app"
>
  <GameContainer />
</div>
`;
